IndexTest.php

```
@@ -0,0 +1,21 @@
+<?php
+
+require './index.php';
+require './indexTestFunctions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class IndexTest extends TestCase
+{
+    public function testMonsterArray_hasMonsters()
+    {
+    //Setup
+        $expectedResult = 'Monsters found!';
+     //Execution
+        $testArray = [1, 3, 4];
+        $result = checkEmpty($testArray);
```

Surely this should return an error because the data you gave to checkEmpty doesn't contain any monsters?